tests/net_inet/ssl_cert.py

```diff
@@ -0,0 +1,65 @@
+try:
+    import usocket as _socket
+except:
+    import _socket
+try:
+    import ussl as ssl
+    import ubinascii as binascii
+except:
+    import ssl
+    import binascii
+
+
+ca_cert_chain = binascii.unhexlify(
```

Comment:
Is this just the Let's Encrypt cert?  Can you please include as a comment the command you used to generate this data.

Reply:
Yes it's this one:
```
1 s:/C=US/O=Let's Encrypt/CN=R3
   i:/C=US/O=Internet Security Research Group/CN=ISRG Root X1
```